Misc/NEWS.d/next/Core and Builtins/2022-08-18-13-47-59.gh-issue-96046.5Hqbka.rst

```diff
@@ -0,0 +1,4 @@
+:c:func:`PyType_Ready` now initializes ``ht_cached_keys`` and performs
```

Comment:
You should move this NEWS entry to the C API category, since PyType_Ready() is consumed by C extension maintainers.